spec/requests/api/v1/conversations/index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe 'GET /api/v1/conversations', type: :request do
+  let(:user)   { create(:user) }
+  let(:user2)  { create(:user) }
+
+  before(:each) do
+    target = create(:target, user_id: user.id)
+    create_list(
+      :target, 2,
+      topic_id: target.topic.id,
+      user_id: user2.id,
+      lat: target.lat,
+      lng: target.lng
+    )
+    create(:conversation, users: [user, user2], user_id: user.id)
+  end
```

Comment:
This is a bit confusing, maybe you can use [traits](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#defining-traits). So you can have like a trait in conversation factory like `with_match` or similar.

Reply:
Done! Thanks for the comments, helps a lot.